Makefile

```diff
@@ -5,14 +5,12 @@ PROVIDER_NAME := gcp
 PROJECT_NAME := provider-$(PROVIDER_NAME)
 PROJECT_REPO := github.com/upbound/$(PROJECT_NAME)
 
-export PROVIDER_NAME
 export TERRAFORM_VERSION := 1.5.5
-export TERRAFORM_PROVIDER_SOURCE := hashicorp/google
 export TERRAFORM_PROVIDER_VERSION := 4.77.0
-export TERRAFORM_PROVIDER_DOWNLOAD_NAME := terraform-provider-google
-export TERRAFORM_PROVIDER_DOWNLOAD_URL_PREFIX := https://releases.hashicorp.com/terraform-provider-google/$(TERRAFORM_PROVIDER_VERSION)
+export TERRAFORM_PROVIDER_SOURCE := hashicorp/google
```

Comment:
Do we still need this variable?

Reply:
Hi @sergenyalcin,
We now use `TERRAFORM_PROVIDER_SOURCE` and the like while extracting the Terraform provider metadata and for generating the `schema.json`. However, it's no longer used at runtime or for downloading the Terraform provider image while building the provider package.